src/index.js

```diff
@@ -8,7 +8,7 @@ const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
     <App />
-  </React.StrictMode>
+  </React.StrictMode>,
```

Comment:
- [ ] Kindy add your redux store to your react App, you need to wrap the App in the Provider componet that you will import from react-redux, and the store will be imported from you redux confug file, this will help you connect redux with your react App

``` 
<Provider store={store}>
    <App />
  </Provider>,
 ```

Reply:
@andersonlebon Done